.github/PULL_REQUEST_TEMPLATE/pull_request_template.md

```diff
@@ -0,0 +1,37 @@
+<!--
+    Thank you for your contribution. Please use this template to help guide
+    you towards preparing a PR that fullfills our mereability  requirements.
+-->
+# Description
+
+<!-- Describe the changes your PR introduces here. -->
+
+# Issues
+
+<!--
+    Link related issues here, it's ok if this is empty but we do recommend that
+    you create issues before working on PRs, issues on internal trackers are
+    fine and need not be linked here.
+-->
+
+# Checklist
+
+<!--
+    Take care of the default items before marking your PR as ready for review,
+    be prepared to add more items.
+-->
+
+* [ ] I updated the version in Chart.yaml
+* [ ] I updated applicable README.md files using  `pre-commit run -a`
+* [ ] I documented any high-level concepts I introduce in `docs/`
```

Comment:
```suggestion
* [ ] I documented any high-level concepts I introduced in `docs/`
```

Reply:
it was intended to be in present tense and I now changed it to "concepts I'm introducing"

Reply:
But how can you use present tense when the sentence starts in past tense? But the version with "concepts I'm introducing" sounds good.